server/aws/mc-lambda.tf

```diff
@@ -0,0 +1,106 @@
+##
+#  Mectrics Collection Lambda
+##
+
+resource "aws_lambda_function" "metrics" {
+  function_name = var.service_name
+  description = var.lambda-description
+  s3_bucket = var.lambda_code
+  s3_key = var.lambda-function-code
```

Comment:
Since we don't have the Github CI setup to package up the lambda, we should deploy the lambda using the same method we did for the Code Deploy Lambda's

https://github.com/cds-snc/covid-alert-server-staging-terraform/blob/e76e7c21fb7b18e74b0ac85828b52ce773fb606f/server/aws/codedeploy.tf#L55